test/system/155-partial-pull.bats

```diff
@@ -157,11 +158,12 @@ function mount_image_and_take_digest() {
 }
 
 @test "zstd chunked does not modify image content" {
+    skip_if_remote "need to mount an image"
```

Comment:
the first skip is part of setup() which is a magic bats function that will be run for each test case so this here is redundant

Reply:
Yes… I was trying to capture “this is one more reason why the test won’t work” but I appreciate that having a redundant skip here might be confusing.

Reply:
Now with a comment.